scripts/lib/install

```diff
@@ -33,6 +33,21 @@ ZULIP_PATH="$(realpath $(dirname $0)/../..)"
 # setup-apt-repo does an `apt-get update`
 "$ZULIP_PATH"/scripts/lib/setup-apt-repo
 
+# Set up SSL certificate
+while getopts ":--domain:--email:--instal-certbot:" o; do
+    case "${o}" in
+        --domain)
+            DOMAIN=$(OPTARG)
+            ;;
+        --email)
+            EMAIL=$(OPTARG)
+            ;;
+        --install-certbot)
```

Comment:
If `--install-certbot` is invoked before `--domain` or `--email`, will it screw up the `setup-certbot`?

Reply:
I think you want to set a variable here and then check that variable in a block after the `getopt` step.

Reply:
Addressed, but not sure if it is correct